snippets/csharp/keywords/FixedKeywordExamples.cs

```diff
@@ -39,6 +40,27 @@ unsafe private static void ModifyFixedStorage()
         }
         // </Snippet1>
 
+        // <SnippetFixedSpan>
+        unsafe private static void FixedSpanExample()
+        {
+            int[] PascalsTriangle = {
+                          1,
+                        1,  1,
+                      1,  2,  1,
+                    1,  3,  3,  1,
+                  1,  4,  6,  4,  1,
+                1,  5,  10, 10, 5,  1
+            };
+
+            Span<int> RowFive = new Span<int>(PascalsTriangle, 10, 5);
+
+            fixed (int* ptrToRow = RowFive)
+            {
+                // Use the numbers 1, 5, 10, 10, 5, 1
```

Comment:
The comment is wrong -- you're taking elements 10 through 14, which are 1,4,6,4,1.
Did you want to do anything with the numbers in the fixed block?